### DISCUSSION TITLE
6.0.0-beta

### DISCUSSION THREAD

**Discussion by derkoe** [porscheinformatik/sonarqube-licensecheck#396] [General]
This pre-release supports the new SonarQube 10.x releases while also supporting 9.5+

BREAKING CHANGES
- This version is not compatible with SonarQube < 9.5

Features
- Compatibility with Sonar 10.x (#375)
- Support for Scala (#352)
- Feature to import SPDX license list (fa68e04422bdd12d05e78c72ee0a49224f6b8741)

<hr /><em>This discussion was created from the release <a href='https://github.com/porscheinformatik/sonarqube-licensecheck/releases/tag/v6.0.0-beta-1'>6.0.0-beta-1</a>.</em>

**Comment by derkoe** [comment]
We now also have a [6.0.0-beta-2](https://github.com/porscheinformatik/sonarqube-licensecheck/releases/tag/v6.0.0-beta-2) with the additional feature:
- Resolve node_modules relative to package.json #380

**Reply by muellerst-hg** [reply]
Successfully tested 6.0.0-beta2 with Sonar 10.3 EE

**Reply by muellerst-hg** [reply]
any dates for a stable release?

**Reply by derkoe** [reply]
We did some testing on our instance this week. Looks good so far - will create the release now
